kronolith/lib/Application.php

```diff
@@ -956,11 +985,19 @@ public function davGetObjects($collection)
             ->getInstance('Horde_Dav_Storage');
 
         $internal = $dav->getInternalCollectionId($collection, 'calendar') ?: $collection;
-        if (!Kronolith::hasPermission($internal, Horde_Perms::SHOW)) {
+        // Handle external collections
+        $exploded = explode(':', $internal, 2);
+        $driverType = null;
+        if ($exploded[0] == "external") {
+            $driverType = "Horde";
+            $internalName = str_replace(':', '/', $exploded[1]);
```

Comment:
Why using a different separator in Kronolith_Calendar_External in the first place?